tasks/get_objects_for_app.yml

```diff
@@ -36,15 +33,6 @@
     - statefulset
     - stream
```

Comment:
should you?

search for "stream" on the code base?

Reply:
Yes I should remove it, it's not used in the application.